.github/ISSUE_TEMPLATE/specreview_checklist.md

```diff
@@ -0,0 +1,65 @@
+# Spec Review Checklist
+
+1. Spec PR
```

Comment:
We shouldn't be defining a PR here...  We need to define the Issue template.  I'll talk with you offline so that we're on the same page...